CHARTER.md

```diff
@@ -0,0 +1,110 @@
+
+# Technical Charter (the “Charter”) for OSV Schema a Series of LF Projects, LLC
+
+**Adopted:**__________
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OSV Schema open source project, which has been established as OSV Schema a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+## 1. Mission and Scope of the Project
+
+a. The mission of the Project is to develop a common format for the sharing of vulnerability information.
```

Comment:
To be consistent with https://ossf.github.io/osv-schema/#purpose, could we use "a standard interchange format for describing vulnerabilities in open source packages." verbatim here?